.github/workflows/codeql.yaml

```diff
@@ -1,3 +1,12 @@
+# Copyright Contributors to the L3AF Project.
+# SPDX-License-Identifier: Apache-2.0
+#
+# For documentation on the github environment, see
+# https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners
+#
+# For documentation on the syntax of this file, see
+# https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+
 name: "CodeQL"
 
```

Comment:
I believe we still need to add the default permissions here to get the check to pass.